server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/decider/WriteLoadConstraintDeciderIT.java

```diff
@@ -68,6 +68,8 @@ protected Collection<Class<? extends Plugin>> getMockPlugins() {
      */
     public void testHighNodeWriteLoadPreventsNewShardAllocation() {
         int randomUtilizationThresholdPercent = randomIntBetween(50, 100);
+        int numberOfWritePoolThreads = randomIntBetween(10, 20);
```

Comment:
I think the lower bound of the randomization should be 2 since that's a realistic setup.